stacks/spica/src/function/pages/add/add.component.scss

```diff
@@ -1,14 +1,33 @@
 :host {
+  .with-sidecar {
+    width: calc(100% - 450px) !important;
+    display: block;
+  }
   mat-toolbar {
     min-height: 250px;
     padding: 0 30px;
     align-items: flex-start;
+    flex-direction: column;
+
     h4 {
       mat-icon,
       span {
         vertical-align: middle;
         margin-left: 5px;
       }
+      margin-bottom: 60px;
+    }
+
+    div.actions {
+      max-width: 100%;
+      align-self: flex-end;
+      overflow-x: scroll;
+
+      -ms-overflow-style: none; /* Internet Explorer 10+ */
+      scrollbar-width: none; /* Firefox */
```

Comment:
These are redundant since we have autoprefixer built into Angular CLI. therefore these lines should be removed or replaced with a standard CSS rule.  

See https://github.com/spica-engine/spica/blob/master/stacks/spica/browserslist